### PR TITLE
Fix cambodia error

### PR DESCRIPTION
**Pull request by pindge**

### CODE REVIEW

File: dev/services/wms/cc-geomedian/ows_cfg.py [hunk 1]
```diff
@@ -344,6 +344,11 @@
                 "horizontal_coord": "x",
                 "vertical_coord": "y",
             },
+            "EPSG:32648": {  # Web Mercator
```
Comment (tom-butler) [inline dev/services/wms/cc-geomedian/ows_cfg.py]: this EPSG isn't web mercator

Comment (pindge) [reply dev/services/wms/cc-geomedian/ows_cfg.py]: i dont know what it is, its part of the comment i copied ows.

```
        "published_CRSs": {
            "EPSG:3857": {  # Web Mercator
                "geographic": False,
                "horizontal_coord": "x",
                "vertical_coord": "y",
            },
            "EPSG:4326": {  # WGS-84
                "geographic": True,
                "vertical_coord_first": True
            },
            "EPSG:3577": {  # GDA-94, internal representation
                "geographic": False,
                "horizontal_coord": "x",
                "vertical_coord": "y",
            },
            "EPSG:3111": {  # VicGrid94 for delwp.vic.gov.au
                "geographic": False,
                "horizontal_coord": "x",
                "vertical_coord": "y",
            },
```

Comment (whatnick) [reply dev/services/wms/cc-geomedian/ows_cfg.py]: It is https://epsg.io/32648 -> UTM Zone 48 N.

Comment (pindge) [reply dev/services/wms/cc-geomedian/ows_cfg.py]: cool, removed `3577` and placed `32648` at the bottom

**Pull request merged**